websites/#/Фугапедия/presence.ts

```diff
@@ -4,250 +4,172 @@ const presence = new Presence({
+		case "h": {
+			presenceData.details = "Смотрит историю статьи";
+			if (!privacyMode) presenceData.state = `«${parseName("article_id")}»`;
 			break;
 		}
-		case "/user": {
-			presenceData.details = "Смотрит профиль";
-			if (!privacyMode) {
-				presenceData.state = `${
-					// A user nickname
-					document.querySelector(
-						"p.text-2xl.text-black.font-medium.ml-4.place-self-center"
-					).firstChild.textContent
-				} (${
-					// A user name
-					document.querySelector(
-						"p.text-lg.text-gray-600.ml-4.place-self-center"
-					).textContent
-				})`;
-				if (showButtons) {
-					presenceData.buttons = [
-						{
-							label: "Перейти к пользователю",
-							url: document.URL
-						}
-					];
-				}
-			}
-
+		case "notifications": {
+			presenceData.details = "Читает уведомления";
+			break;
+		}
+		case "offer": {
+			presenceData.details = "Читает публичную оферту";
 			break;
 		}
-		case "/privacypolicy": {
+		case "privacy": {
 			presenceData.details = "Читает политику";
 			presenceData.state = "конфиденциальности";
-
 			break;
 		}
-		case "/termsofuse": {
-			presenceData.details = "Читает условия";
-			presenceData.state = "использования";
-
+		case "s": {
+			const query = parseName(3);
+			if (!privacyMode && query) {
+				presenceData.details = "Ищет что-то по запросу";
+				presenceData.state = `«${query}»`;
+			} else presenceData.details = "Ищет что-то";
 			break;
 		}
-		case "/ad": {
-			presenceData.details = "Читает условия";
-			presenceData.state = "размещения рекламы";
-
+		case "subscribe": {
+			presenceData.details = "Собирается купить подписку";
 			break;
 		}
-		case "/account": {
-			if (document.location.hash === "#admintools")
-				presenceData.details = "Что-то делает в админ-панели";
-			else presenceData.details = "Редактирует свой аккаунт";
-
+		case "t": {
+			presenceData.details = "Смотрит категорию";
+			if (!privacyMode) presenceData.state = `«${parseName(3)}»`;
 			break;
 		}
-		case "/sub": {
-			presenceData.details = "Собирается купить подписку";
+		case "tags": {
+			presenceData.details = "Смотрит все категории";
 			break;
 		}
-		case "/upload": {
-			presenceData.details = "Загружает изображение";
+		case "termsofuse": {
+			presenceData.details = "Читает условия использования";
 			break;
 		}
-		case "/api.php": {
-			presenceData.details = "Читает документацию API";
+		case "u": {
+			presenceData.details = "Смотрит профиль пользователя";
+			if (!privacyMode) {
+				if (splittedPathname[2] === "me") {
+					presenceData.details = "Смотрит свой профиль";
+					presenceData.state = document.querySelector(
+						"body > div > div > main > div > div > div > div > div > h2.font-serif-text"
+					).textContent;
+				} else presenceData.state = parseName("profile_user_id");
+				presenceData.largeImageKey = document.querySelector<HTMLImageElement>(
+					"body > div > div > main > div > div > div > div > div > img"
+				).src;
+				presenceData.smallImageKey = "logo";
```

Comment:
these should be swapped around

Reply:
pfp can't be in the large image?

Reply:
Large image should always be the service logo

Reply:
Large image should always be the service logo

Reply:
But some presences use it not only for the logo. And I didn't find any info that it should always be the service logo.